rotkehlchen/chain/ethereum/modules/eth2/eth2.py

```diff
@@ -344,7 +344,11 @@ def get_performance(
                 from_ts_ms=ts_sec_to_ms(profit_from_ts),
                 to_ts_ms=ts_sec_to_ms(profit_to_ts),
             )
-            data['apr'] = ((YEAR_IN_SECONDS * validator_sum) / (profit_to_ts - profit_from_ts)) / time_weighted_avg  # noqa: E501
+            if time_weighted_avg != ZERO:
+                data['apr'] = ((YEAR_IN_SECONDS * validator_sum) / (profit_to_ts - profit_from_ts)) / time_weighted_avg  # noqa: E501
+            else:
+                data['apr'] = ZERO
```

Comment:
```suggestion
            data['apr'] = (
                ((YEAR_IN_SECONDS * validator_sum) / (profit_to_ts - profit_from_ts)) / time_weighted_avg  # noqa: E501
                if time_weighted_avg != ZERO else ZERO
            )
```
nitpick, but I think this is a bit cleaner than having `data['apr'] =` twice